packages/frontend/src/components/DashboardTiles/DashboardMinimalDownloadCsv.tsx

```diff
@@ -1,43 +1,151 @@
-import { isField, type ApiChartAndResults } from '@lightdash/common';
+import {
+    getCustomLabelsFromTableConfig,
+    getHiddenTableFields,
+    getItemMap,
+    isField,
+    isTableChartConfig,
+    pivotQueryResults,
+    type ApiChartAndResults,
+} from '@lightdash/common';
 import { Menu } from '@mantine/core';
 import { IconTableExport } from '@tabler/icons-react';
 import { stringify } from 'csv-stringify/browser/esm';
 import { useCallback, type FC } from 'react';
+import useApp from '../../providers/App/useApp';
+import useTracking from '../../providers/Tracking/useTracking';
+import { EventName } from '../../types/Events';
 import MantineIcon from '../common/MantineIcon';
 
+const pivotResultsAsCsv = (
```

Comment:
ugh, yeah, I believe this is because we needed to export pivot csv charts on embedded dashoards..